sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/statsbeat/_statsbeat_metrics.py

```diff
@@ -16,42 +24,29 @@
     _REQ_THROTTLE_NAME,
 )
 from azure.monitor.opentelemetry.exporter.statsbeat._state import (
-    _REQUESTS_LOCK,
+    _REQUESTS_MAP_LOCK,
     _REQUESTS_MAP,
 )
 
 
+_AIMS_URI = "http://169.254.169.254/metadata/instance/compute"
+_AIMS_API_VERSION = "api-version=2017-12-01"
+_AIMS_FORMAT = "format=json"
+
 _ENDPOINT_TYPES = ["breeze"]
 _RP_NAMES = ["appsvc", "functions", "vm", "unknown"]
 
 _HOST_PATTERN = re.compile('^https?://(?:www\\.)?([^/.]+)')
 
 # cSpell:disable
 
-# pylint: disable=unused-argument
-# pylint: disable=protected-access
-def _get_success_count(options: CallbackOptions) -> Iterable[Observation]:
```

Comment:
I understand that only success_count is moved because it needs to utilize the counter, but I'm wondering if for simplicity we should just move all of the methods into the class

Reply:
Not an approval blocker. Just a thought

Reply:
I think you're right. Moved all of the methods into the class.